admin/resources/js/adminConstants.ts

```diff
@@ -1,3 +1 @@
-export const BASE_URL =
-  process.env.ADMIN_APP_URL ?? "http://localhost:8000/admin";
 export const ADMIN_APP_DIR = process.env.ADMIN_APP_DIR ?? "admin";
```

Comment:
I am getting an ESLint error for this line since there is only a single `const` now.
Prefer default export: [import/prefer-default-export](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/prefer-default-export.md).

Reply:
@mnigh could you tell me how do you run the linting on the repo?

Reply:
nvm . got it

Reply:
I think this and talentSearchConstants are a situation where I would ignore the rule, rather than add the default export 😄.  If I was trying to use the default export (which I generally avoid) of a file like this, I would expect the result to be a dictionary, not a single value. 
So actually, maybe default export an object `{ ADMIN_APP_DIR }` if we really want to satisfy the rule 🤔.